tools/build_chain.sh

```diff
@@ -1408,7 +1442,7 @@ for line in ${ip_array[*]};do
             fi
 
             if [ -n "$guomi_mode" ]; then
-                gen_node_cert_gm ${output_dir}/gmcert/agency ${node_dir} >${logfile} 2>&1
+                gen_node_cert_gm ${output_dir}/gmcert/${agency_array[${server_count}]} ${node_dir} #>>"${logfile}"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5331982)